src/Microsoft.Azure.SignalR/ServiceOptionsExtensions.cs

```diff
@@ -14,6 +16,14 @@ public static void Validate(this ServiceOptions options)
             {
                 throw new AzureSignalRInvalidServiceOptionsException("MaxPollIntervalInSeconds", "[1,300]");
             }
+
+
+            if (!string.IsNullOrEmpty(options.ApplicationName) && !Regex.IsMatch(options.ApplicationName, "^[a-zA-Z][a-zA-Z0-9_]*$"))
+            {
+                throw new AzureSignalRInvalidServiceOptionsException("ApplicationName", "should start with alphabetic characters and only contain alpha-numeric characters or underscore.");
```

Comment:
>should start with alphabetic characters and only contain alpha-numeric characters or underscore. [](start = 89, length = 96)

Base message is formatted like "Property '{propertyName}' value should be {validScope}." [https://github.com/Azure/azure-signalr/blob/dev/src/Microsoft.Azure.SignalR.Common/Exceptions/AzureSignalRInvalidServiceOptionsException.cs#L13](https://github.com/Azure/azure-signalr/blob/dev/src/Microsoft.Azure.SignalR.Common/Exceptions/AzureSignalRInvalidServiceOptionsException.cs#L13).

Update the sentence to make it smooth.